API-Automation/tasks/api-tasks.js

```diff
@@ -1,173 +1,152 @@
 import axiosInstance from "../support/axiosConfig.js";
-import {
-  createUserPath,
-  createUserWithListPath,
-  deleteUserInvalidUsernamePath,
-  getUserPath,
-  userLoginPath,
-  userLoginPathUsername,
-  deleteUserPath,
-  userLogoutPath,
-  addNewPetPath,
-  editPetPath,
-  findByStatusPath,
-  findPetByIdPath,
-  deletePetPath,
-  createOrderPath,
-  getPurchaseByPetIdPath,
-  getInventoryPath,
-  deleteOrderByIdPath
-} from "../data/commonData.js";
+import { config } from "../../config.js";
 
-export async function getUserByUsername() {
+
+export async function getUserByUsername(username) {
   try {
-    return await axiosInstance.get(getUserPath.path);
+    return await axiosInstance.get(config.user.getUser.url(username));
   } catch (error) {
-    console.error(error);
+    console.error("Error message: ", error.response.data.message);
   }
 }
 
-export async function createUser(newUserData) {
+export async function createUser(data) {
   try {
-    return await axiosInstance.post(createUserPath.path, newUserData);
+    return await axiosInstance.post(config.user.createUser.url, data);
   } catch (error) {
-    console.error(error);
+    console.error("Error message: ", error.response.data.message);
   }
 }
 
-export async function createUserWithList(createUserWithListData) {
+export async function createUserWithList(data) {
   try {
     return await axiosInstance.post(
-      createUserWithListPath.path,
-      createUserWithListData
+      config.user.createUserWithList.url,
+      data
     );
   } catch (error) {
-    console.error(error);
+    console.error("Error message: ", error.response.data.message);
   }
 }
 
-export async function userLogin() {
+export async function userLogin(username, password) {
   try {
-    return await axiosInstance.get(userLoginPath.path);
+    return await axiosInstance.get(config.user.userLogin.url(username, password));
   } catch (error) {
-    console.error(error);
+    console.error("Error message: ", error.response.data.message);
   }
 }
 
-export async function loginWithUsernameOnly() {
+export async function loginWithUsernameOnly(username) {
```

Comment:
See, it is easier when we know what is the data, i.e. username

Reply:
Yes, I see. It is more intuitive when we know what the data is. __